style/js/shield_defs.js

```diff
@@ -140,8 +140,17 @@ export function loadShields(shieldImages) {
 
   shields["US:US:Truck"] = banneredShield(shields["US:US"], ["TRK"]);
```

Comment:
We found that the sole Truck route was really "Truck Bypass" around Warrenton, GA
```suggestion
  shields["US:US:Truck:Bypass"] = banneredShield(shields["US:US"], ["TRK", "BYPASS"]);
```

Reply:
Do you mean to be removing `US:US:Truck` here?

Reply:
Nope, I edited the US:US section when I meant to edit the US:DE section.  Thanks for catching this.